packages/core/src/components/popover-core/popover-core.tsx

```diff
@@ -32,6 +33,9 @@ export class TdsPopoverCore {
   /** Decides if the component should be visible from the start. */
   @Prop() defaultShow: boolean = false;
 
+  /** Whether the popover should animate when being opened/closed or not */
+  @Prop() animation: 'none' | 'fade' | string = 'none';
+
```

Comment:
As we talked: We need to "bridge" this rule in popover-menu, popover-canvas and tooltip component. Double check tooltip as it might already have a animation, so maybe it should be skipped.